executor/point_get.go

```diff
@@ -49,26 +49,36 @@ func (b *executorBuilder) buildPointGet(p *plannercore.PointGetPlan) Executor {
 		return nil
 	}
 
-	startTS, err := b.getSnapshotTS()
+	if p.Lock && !b.inSelectLockStmt {
+		b.inSelectLockStmt = true
+		defer func() {
+			b.inSelectLockStmt = false
+		}()
+	}
+
+	snapshotTS, err := b.getSnapshotTS()
 	if err != nil {
-		b.err = err
```

Comment:
Why is it removed?

Reply:
I think it may be deleted accidentally? I will add it back.